lib/simple-seeder.js

```diff
@@ -156,12 +160,15 @@ module.exports = class SimpleSeeder {
         }
       }
 
-      for await (const e of snap.createReadStream()) {
+      for await (const e of snap.createReadStream(contentEncoding)) {
         if (e.value.type === 'list') continue // List of lists is not supported
 
         await this.put(e.key, { ...e.value, external: false })
       }
 
+      const entry = await snap.get(ALLOWED_PEERS, metadataEncoding)
+      info.userData.allowedPeers = entry ? entry.value : null
```

Comment:
Security wise, let's move this above the `createReadStream`

So it prunes first, then we update allowed peers, and then we put (add/change)